### DISCUSSION TITLE
Where is the Wineskin Winery source code?

### DISCUSSION THREAD

**Discussion by ajmas** [The-Wineskin-Project/WineskinServer#378] [Q&A]
Is Wineskin Winery open source? I was looking through the project and could not see any references to the launcher source code?

I could only find the [old source code](https://sourceforge.net/p/wineskin/code/ci/master/tree/) on SourceForge

**Comment by Gcenx** [comment]
I've not pushed the current sources anywhere public for like 3? years now, you can find an older version https://github.com/vitor251093/wineskin/tree/Unoffical_Wineskin

**Reply by nevack** [reply]
Any specific reason behind this?
Licensing issues?
Honestly, I would like to contribute too!

**Reply by sonatique** [reply]
me too
